### PR TITLE
Update menu.css

### PR DESCRIPTION
**Pull request by KoniKodes**
line 152 changes drop-down sub-menu to white

### CODE REVIEW

File: interface/main/tabs/css/menu.css [hunk 1]
```diff
@@ -149,6 +149,7 @@ and open the template in the editor.
     padding-left: 20px;  
     padding-bottom : 0px; 
     font-weight : 600;
+    background-color: #fff; // changes drop-down sub-menu to white
 }
```
Comment (muarachmann) [inline interface/main/tabs/css/menu.css]: @KoniKodes seems to do the trick will test this in a bit.

Comment (KoniKodes) [reply interface/main/tabs/css/menu.css]: Thank you. It worked in the Inspector...

**Comment by muarachmann** [comment]
@KoniKodes do you have the latest version of the codes. Cause i don't reproduce your issue locally

**Comment by muarachmann** [comment]
I dont think the EHR Docs site has the latest codes thats my pb...

**Comment by KoniKodes** [comment]
It doesn't make sense to give us a site that does not have the latest if they cannot return the site they took down.

**Comment by muarachmann** [comment]
@KoniKodes can you place the link to the site here so i verify some stuffs

**Comment by KoniKodes** [comment]
@muarachmann 
https://ehr.librehealth.io/interface/login/login.php?site=docs
username: admin
password: documentationisawesome

Takes you to https://ehr.librehealth.io/interface/main/tabs/main.php?url=TAB_ONE_DEFAULT

I am having the issues from this site...

Your help is greatly appreciated.

**Comment by aethelwulffe** [comment]
@muarachmann did you get a second check on this?  I will test today myself.

**Comment by muarachmann** [comment]
@aethelwulffe this is on the doc site not on this master branch. I dont think that site fully reflects the master branch we have here

**Comment by KoniKodes** [comment]
@aethelwulffe @muarachmann 
It is my understanding that the EHR Docs site is supposed to reflect the master branch. 
It is reserved for us to add users/patients as needed for the Documentation Manual screen shots.
However, we are not able to do what we could previously. Some of those issues are discussed on the Chat in the #outreachy and #outreachy submissions channels. I've also attempted to reach out to the #devs channel in the Chat. 

Any help you could give us would be greatly appreciated.

**Comment by aethelwulffe** [comment]
Waiting too long on this.  Merging without second review.  This is hurting us with fewer reviewers.

**Comment by KoniKodes** [comment]
Thank you so much!

**Pull request merged**